.github/workflows/publish-internal.yml

```diff
@@ -0,0 +1,68 @@
+name: "Publish to internal"
```

Comment:
This workflow was created separately because we need to run tests and publish all in isolated streams. This prevents failing tests from one adapter blocking the release of another adapter.